packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.ts

```diff
@@ -0,0 +1,863 @@
+ * @part answer-list-item - The generated answer list item elements for both ordered and unordered lists.
+ * @part answer-ordered-list - The generated answer ordered list elements.
+ * @part answer-paragraph - The generated answer paragraph elements.
+ * @part answer-quote-block - The generated answer quote block elements.
+ * @part answer-unordered-list - The generated answer unordered list elements.
+ * @part answer-strong - The generated answer strong text elements.
+ * @part answer-table - The generated answer table elements.
+ * @part answer-table-container - The generated answer table container elements.
+ * @part answer-table-content - The generated answer table content cell elements.
+ * @part answer-table-header - The generated answer table header cell elements.
+ *
+ * @part citation - The link that allows the user to navigate to the item.
+ * @part citation-popover - The pop-up that shows an item preview when the user hovers over the citation.
+ */
+@customElement('atomic-generated-answer')
+@bindings()
+@withTailwindStyles
+export class AtomicGeneratedAnswer
+  extends LitElement
+  implements InitializableComponent<Bindings>
+{
+  static styles = [atomicGeneratedAnswerStyles];
+  private static readonly propsSchema = new Schema({
+    maxCollapsedHeight: new NumberValue({
+      min: 9,
+      max: 32,
+      required: false,
+    }),
+  });
+
+  private readonly DEFAULT_COLLAPSED_HEIGHT = 16;
+  private readonly MAX_COLLAPSED_HEIGHT = 32;
+  private readonly MIN_COLLAPSED_HEIGHT = 9;
+  private readonly REQUIRED_FIELDS_TO_INCLUDE_IN_CITATIONS = ['filetype'];
+
+  private resizeObserver?: ResizeObserver;
+  private fullAnswerHeight?: number;
+  private storage: SafeStorage = new SafeStorage();
+  private storedData!: GeneratedAnswerData;
+  private modalRef?: HTMLAtomicGeneratedAnswerFeedbackModalElement;
+
+  /**
+   * Whether to render a toggle button that lets the user hide or show the answer.
+   */
+  @property({
+    type: Boolean,
+    attribute: 'with-toggle',
+    converter: booleanConverter,
+  })
+  withToggle = false;
+
+  /**
+   * Whether to allow the answer to be collapsed when the text is taller than the specified `--atomic-crga-collapsed-height` value (16rem by default).
+   */
+  @property({type: Boolean, converter: booleanConverter})
+  collapsible = false;
+
+  /**
+   * The maximum height (in rem units) of the answer when collapsed.
+   */
+  @property({type: Number, attribute: 'max-collapsed-height'})
+  maxCollapsedHeight = this.DEFAULT_COLLAPSED_HEIGHT;
+
+  /**
+   * The unique identifier of the answer configuration to use to generate the answer.
+   */
+  @property({type: String, attribute: 'answer-configuration-id'})
+  answerConfigurationId?: string;
+
+  /**
+   * A list of fields to include with the citations used to generate the answer.
+   */
+  @property({type: String, attribute: 'fields-to-include-in-citations'})
+  fieldsToIncludeInCitations?: string;
```

Comment:
Alas, @louis-bompart has made some changes to the original Stencil component yesterday, and the prop should now also support a string[]

See https://github.com/coveo/ui-kit/pull/6738/files

There were also PR review comments applied later from what I can see:

https://github.com/coveo/ui-kit/pull/6739/files